spark/dl/src/test/scala/com/intel/analytics/bigdl/keras/nn/GaussianDropoutSpec.scala

```diff
@@ -1,8 +1,42 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 package com.intel.analytics.bigdl.keras.nn
 
-/**
-  * Created by intel on 2018/1/25.
-  */
-class GaussianDropoutSpec {
+import com.intel.analytics.bigdl.keras.KerasBaseSpec
+import com.intel.analytics.bigdl.nn.keras.GaussianDropout
+import com.intel.analytics.bigdl.nn.keras.{Sequential => KSequential}
+import com.intel.analytics.bigdl.tensor.Tensor
+import com.intel.analytics.bigdl.utils.Shape
+
+class GaussianDropoutSpec extends KerasBaseSpec {
+
+  "GaussianDropout" should "generate the correct outputShape" in {
+    val seq = KSequential[Float]()
+    val gaussiandropout = GaussianDropout[Float](0.5, inputShape = Shape(3))
+    seq.add(gaussiandropout)
+    gaussiandropout.getOutputShape().toSingle().toArray should be(Array(-1, 3))
+  }
 
+  "GaussianDropout forward and backward" should "work properly" in {
+    val seq = KSequential[Float]()
+    val input = Tensor[Float](Array(2, 28, 28, 1)).rand()
+    val gaussiandropout = GaussianDropout[Float](0.6, inputShape = Shape(3))
```

Comment:
Input tensor is of shape (2, 28, 28, 1) but in the layer you specify inputShape=Shape(3)??